.rubocop_todo.yml

```diff
@@ -1,3 +1,5 @@
+require: rubocop-rspec
```

Comment:
I don't think so, I think the rubocop_todo.yml is the things we thing ought to get fixed.